app/assets/stylesheets/govuk_publishing_components/components/helpers/_variables.scss

```diff
@@ -11,7 +11,7 @@ $gem-spacing-scale-7: 50px;
 $gem-spacing-scale-8: 60px;
 
 $gem-text-colour: $text-colour;
-$gem-secondary-text-colour: $grey-1;
+$gem-secondary-text-colour: govuk-colours("dark-grey");
```

Comment:
[Aside: This'll compile just fine because Sass will assume that `govuk-colours` is a native CSS construct (like e.g. `background-image`) – this sort of thing can easily slip in, so we've written [a test to try and catch it](https://github.com/alphagov/govuk-frontend/blob/08b408840deabc40cb9d2438b0d19862cda41f05/src/govuk/all.test.js#L116-L155)]

Reply:
Ha, good found, and even better test, but it's a bit hard to replicate it in the world of Jasmine. The mixin failed silently and it had no visual effects since this variable is no longer used (was replaced by `$govuk-secondary-text-colour` all over the gem. This should be removed now.